src/ai/backend/client/cli/admin/quota_scope.py

```diff
@@ -0,0 +1,202 @@
+import sys
+import uuid
+from typing import Optional
+
+import click
+
+from ai.backend.cli.types import ExitCode
+from ai.backend.client.session import Session
+
+from ....common.types import QuotaScopeID
```

Comment:
We discourage relative importing components outside current package, where package scope is determined by the wheel artifact - that says, you can not relative import variables or functions provided by file which in production will be installed by another Python package. 
```suggestion
from ai.backend.common.types import QuotaScopeID
```